proxies/shared/policies/RaiseFault.405NotAllowed.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
The proxy documentation needs updating with this flow.

Reply:
Need to make these updates here -> `specification/communications-manager.yaml`